WooCommerce/Classes/ViewModels/MainTabViewModel.swift

```diff
@@ -93,8 +95,7 @@ final class MainTabViewModel {
         listenToReviewsBadgeReloadRequired()
         retrieveShouldShowReviewsBadgeOnHubMenuTabValue()
 
-        listenToNewFeatureBadgeReloadRequired()
-        retrieveShouldShowNewFeatureBadgeOnHubMenuTabValue()
+        tapToPayBadgePromotionChecker.$shouldShowTapToPayBadges.share().assign(to: &$shouldShowNewFeatureBadgeOnHubMenuTab)
```

Comment:
Same as `HubMenuViewModel`: Should we wrap this line in a method? Sort of "bind/observe badge changes"?